service/process_wrapper.py

```diff
@@ -13,10 +13,18 @@
 import websockets
 from websocket_requests.job_commands import JobStdout, JobStderr, JobReturnCode
 
-logging.basicConfig()
+logging.basicConfig(level=logging.DEBUG)
```

Comment:
Unfortunately, we can't use the global logging config because this is spawned in a separate process.